docs/versioning.md

```diff
@@ -0,0 +1,46 @@
+# Versioning
+
+
+The airo-mono repo uses a single global version using a `YYYY.MM.N version scheme`.
+Compatability between the packages is only guaranteed between identical versions, so you should use the same version of all packages.
+
+The single point of truth for the version is in the `.bumpversion.toml` file.
+
+## Scheme
+We use a [Calender Version](https://calver.org/) scheme as follows:
+
+```
+YYYY.MM.N
+```
+
+where YYYY.MM is the year and month in which the release is made and N is the MICRO part used to distinguish between multiple releases in the same month. No semantic meaning can be attached to the micro parts and breaking changes might occur in each release for now.
+
+## Versioning strategy
+The main branch ('our trunk') will always live at last released version. All development builds will hence have the **previous** release as base.
```

Comment:
Do I understand correctly that this is how we should develop:
1. first develop you features/bug fixes on a branch, but leave the version at the old one
2. just before (or after?) merge branch into main: bump version of branch (or main?)